tracim/tracim/lib/webdav/sql_domain_controller.py

```diff
@@ -2,6 +2,9 @@
 
 from tracim.lib.user import UserApi
 
+class DigestAuthNotImplemented(Exception):
+    pass
+
 class TracimDomainController(object):
```

Comment:
Do we really need the TracimDomainController which implements nothing except a raise of DigestAuthNotImplemented ?

Reply:
Yes, TracimDomainController is needed. It does Webdav->Tracim User API  convertion to allow authentification. (see isRealmUser and authDomainUser)